libs/components/forms/src/lib/modules/input-box/input-box.component.ts

```diff
@@ -286,6 +288,11 @@ export class SkyInputBoxComponent
     this.#changeRef.markForCheck();
   }
 
+  public setHintTextScreenReaderOnly(hide: boolean): void {
+    this.hintTextRemoved = hide;
```

Comment:
This is minor - but I could see us having a future need to completely remove the hint text via an `ngIf`. Should we call this variable `hintTextScreenReaderOnly` or `hintTextSrOnly` so that we can use `Removed` in the future if we ever need to truly remove it?

Reply:
changed